container/Dockerfile.docs

```diff
@@ -29,26 +29,26 @@ WORKDIR /workspace/dynamo
 
 ENV VIRTUAL_ENV=/workspace/dynamo/.venv
 RUN uv venv $VIRTUAL_ENV --python 3.12 && \
-    uv pip install ablog \
-      attrs  \
-      breathe \
-      docutils \
-      exhale \
-      httplib2 \
-      ipython \
-      myst-nb \
-      nbclient \
-      nbsphinx \
-      nvidia-sphinx-theme \
-      sphinx \
-      sphinx-book-theme \
-      sphinx-copybutton \
-      sphinx-design \
-      sphinx-prompt \
-      sphinx-sitemap \
-      sphinx-tabs \
-      sphinxcontrib-bibtex \
-      sphinxcontrib-mermaid
+    uv pip install ablog==0.11.12 \
```

Comment:
Should the version come from a requirements file? Then use -r? It might be easier to maintain